clerk/clients.go

```diff
@@ -0,0 +1,49 @@
+package clerk
+
+import "fmt"
+
+type ClientsService service
+
+type ClientResponse struct {
+	Object              string  `json:"object"`
+	ID                  string  `json:"id"`
+	LastActiveSessionID *string `json:"last_active_session_id"`
+	SignInAttemptID     *string `json:"sign_in_attempt_id"`
+	SignUpAttemptID     *string `json:"sign_up_attempt_id"`
+	Ended               bool    `json:"ended"`
+}
+
+func (s *ClientsService) ListAll() ([]ClientResponse, error) {
+	clientsUrl := "clients"
+	req, _ := s.client.NewRequest("GET", clientsUrl)
+
+	var clients []ClientResponse
+	_, err := s.client.Do(req, &clients)
+	if err != nil {
+		return nil, err
+	}
+	return clients, nil
+}
+
+func (s *ClientsService) Read(clientId string) (*ClientResponse, error) {
+	clientUrl := fmt.Sprintf("clients/%v", clientId)
```

Comment:
🙃 Do we still need this? If no please remove.

Reply:
Not needed for the verification, but I can still see this operation in the server API. 
Are we planning to remove it?

Reply:
I was referring to line 29 that prints the ClientURL

Reply:
Ah, that's not a print. 
It's for formatting the string, basically similar to `"clients/" + clientId`.

Reply:
Ouch, that's right. My bad. Apologies.

Reply:
@agis 👍 Sounds good.